src/components/DateInput.jsx

```diff
@@ -180,6 +182,7 @@ class DateInput extends React.PureComponent {
       disabled,
       required,
       readOnly,
+      noAriaLabel,
```

Comment:
instead of `noAriaLabel`, I would recommend something like `hasExternalLabel`, which more clearly indicates that the reason why you wouldn't set the `aria-label` is because there's an external label that's being used. It's also clearer for people who don't understand what ARIA is or why they'd not want it.

Reply:
I like that better, I'll change it.